device/transport/mqtt/devdoc/mqtt_requirements.md

```diff
@@ -266,3 +266,34 @@ The `getTwinReceiver` method creates a `MqttTwinReceiver` object for the twin re
 **SRS_NODE_DEVICE_MQTT_18_007: [** If a twin receiver for this endpoint previously existed, the `getTwinReceiver` method should return the preexisting `MqttTwinReceiver` object as the second parameter of the `done` function with null as the first parameter. **]**
 
 
+### enableC2D
+
+**SRS_NODE_DEVICE_MQTT_16_047: [** `enableC2D` shall connect the MQTT connection if it is disconnected. **]**
```

Comment:
>MQTT connection if it is disconnected. [](start = 65, length = 38)

What's the contract for SUB/SUBACK.  Does this call done before or after the SUBACK is received?  Same for enableMethods.  I can't remember if there's an UNSUBACK, but this question may also apply to disable*